### DISCUSSION TITLE
watch timeout

### DISCUSSION THREAD

**Discussion by benthien** [foundation/proton#15] [General]
I set watch: true in esbuild.config.js. When I ran proton build I got the following output

![watch_timeout](https://user-images.githubusercontent.com/12056197/148900735-3c48a52c-0e20-429b-a64b-3f8351dddaac.png)

**Comment by joeworkman** [comment]
Why not use the built in watch command in proton?

**Reply by benthien** [reply]
When I run proton watch I get

![proton_watch](https://user-images.githubusercontent.com/12056197/148999323-15573681-5a66-4332-8bfe-ed37c5c7c2e9.png)

**Comment by joeworkman** [comment]
Looks like I need to document watch. Have a read through the last release notes. https://github.com/foundation/proton/releases/tag/v0.5.0

**Reply by benthien** [reply]
I ran `yarn add browser-sync chokidar -D` prior to running proton watch. In proton.yml I have
npmBuild: yarn build
devserver: browsersync
Is there something else I need to make it work?
